src/test/java/Homework19.java

```diff
@@ -0,0 +1,32 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class Homework19 extends BaseTest{
+
+    @Test(enabled = true)
+    public void testDeletePlaylist()throws InterruptedException{
+        //
+        String namePl   = "MyPlaylist";
+        String selector = "//section[@id='playlists']//li//a[contains(text(),'"+namePl+"')]";
+
+        login();
+
+        try{
+            WebElement MyPlaylist = findElement("//section[@id='playlists']//li//a[contains(text(),'"+namePl+"')]");
```

Comment:
For variables, the Java naming convention is to always start with a lowercase letter and then capitalize the first letter of every subsequent word.

For example  WebElement **myPlaylist**

Reply:
ok, variables was renamed